experimental/sev_guest/src/msr.rs

```diff
@@ -257,30 +258,41 @@ pub struct RegisterGhcbGpaResponse {
     ghcb_gpa: u64,
 }
 
+#[derive(Debug, Snafu)]
+pub enum RegisterGhcbGpaError {
+    InvalidResponse,
+    GhcbLocationNotAccepted,
+    GhcbResponseLocationNotMatchingRequest { addr: u64 },
```

Comment:
Out of curiosity: which address do we send back here?

Reply:
It's the address from the response. I've updated the name of the variable to be more descriptive.